### ISSUE TITLE
Getting different ripple color effect

### ISSUE THREAD

**Issue by nielszandsteeg** [mdbootstrap/mdb-ui-kit#423]
Hi,

So as you can read I'm getting a different ripple color effect compared to the demo. As far as I know I'm using the same css and js files.

Demo:
![image](https://cloud.githubusercontent.com/assets/8080946/6143605/09db6ce0-b1cc-11e4-8f57-5fd4e8044af4.png)

Mine (same background-color and color):
![image](https://cloud.githubusercontent.com/assets/8080946/6143620/4747446e-b1cc-11e4-8141-7c248a8cc033.png)

I know about the function in the ripples.js, who is returning the color. 
window.getComputedStyle($element[0]).color

I would also like getting more a light blue color in place of white. 

Thanks!

**Comment by GradyD** [comment]
Hey,

I am unable to duplicate this, could you create a code pen for this?

**Comment by nielszandsteeg** [comment]
Sorry I am not able to duplicate it. But I can give you some more info. 

I was implementing the sass files in my Visual Studio project. And I saw there was something different between my ripples.scss and the ripples.css in the dist folder. 

Namely,

.ripple.ripple-on {
  transition: opacity 0.15s ease-in 0s, transform 0.5s cubic-bezier(0.4, 0, 0.2, 1) 0.1s;
  opacity: 0.1;
}

And..

.ripple.ripple-on {
    transition: opacity 0.15s ease-in 0s, transform 0.5s cubic-bezier(0.4, 0, 0.2, 1) 0.1s;
    opacity: 1;
}

I did the presumption that css values (in this case: opacity) where the same, but it isn't. Now I have a similar problem with the focus style of the inputs. I haven't found the different yet. The problem is that I havent any focus style at all.

Maybe you've got an idea!?
